tutorial/setup/server-management.md

```diff
@@ -0,0 +1,21 @@
+#Basic Server User Management
+---
+Currently as of v0.19.3 there is no graphical interface for user management from the server level thus you have to use python scripts for user management.
+
+## List all usernames on server  
+'''
+<python>
+[ u.name for u in User.select() ]
+</python>
+'''  
```

Comment:
You can add syntax highlighting with:


<pre>```python
def x(i):
    pass
```</pre>

e.g.:
```python
def x(i):
    pass
```

Reply:
Ah wasn't sure how the interpreter read the .md files, they're all a little different I went with the supposedly "global" option